### ISSUE TITLE
Admin panel doesn't recognize a drop down is not empty if it's left on its first value

### ISSUE THREAD

**Issue by hanrelan** [ash-project/ash_admin#65]
**Describe the bug**
When you have a dropdown from, for example:
```elixir
    attribute :type, :atom,
      constraints: [one_of: [:a, :b, :c]],
      allow_nil?: false
```
In the admin panel, if you don't change the dropdown and leave it on 'a', you get a "type field is required" error when trying to save.

Changing type to 'b' and then back to 'a' fixes it.

**Expected behavior**
It allows you to save while 'a' is set without having to change it first.

**Runtime**
 - Elixir version: 1.15
 - Erlang version: 25
 - OS: Mac OS
 - Ash version: 2.13.3
 - any related extension versions: ash_admin: 0.9.4

**Comment by joshprice** [comment]
Should be fixed by #69, could you please confirm?

**Comment by hanrelan** [comment]
Confirmed fixed! Thanks!
